SpreadCheetah.SourceGenerator.SnapshotTest/Tests/WorksheetRowGeneratorCellValueConverterTests.cs

```diff
@@ -0,0 +1,128 @@
+using SpreadCheetah.SourceGenerator.SnapshotTest.Helpers;
+using SpreadCheetah.SourceGenerators;
+
+namespace SpreadCheetah.SourceGenerator.SnapshotTest.Tests;
+
+public class WorksheetRowGeneratorCellValueConverterTests
```

Comment:
First of all, it's awesome that you have written tests as well 👍
These follow the same style as some of the older tests I've written. Going forward I want to have these kind of tests more self-contained, meaning that the whole code is part of each test, and that they don't reference test classes from elsewhere. The tests in `CellStyleTests.cs` are an example of this.

Reply:
done